pyanaconda/ui/gui/spokes/source.py

```diff
@@ -1328,6 +1328,8 @@ def on_addRepo_clicked(self, button):
         self._repoSelection.select_iter(itr)
         self._repoEntryBox.set_sensitive(True)
 
+        self._repoURLChecks[repo].update_check_status()
```

Comment:
In the Python 3 ``repo`` is not hashable and we use ``repo.name`` as the key. Just please sync with @M4rtinK to make sure whatever goes last has this glitch fixed.